spark/src/main/java/com/nflabs/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -282,7 +292,9 @@ public int getProgress(InterpreterContext context) {
 
   @Override
   public Scheduler getScheduler() {
```

Comment:
Can you make `getScheduler()` conditionally return `getSparkInterpreter().getScheduler();` or `SchedulerFactory.singleton().createOrGetParallelScheduler(...)` depending on newly added property `concurrentSQL()` ?